packages/client/components/NewJiraIssueInput.tsx

```diff
@@ -0,0 +1,195 @@
+  display: 'flex',
+  flexDirection: 'column'
+})
+
+const Item = styled('div')({
+  backgroundColor: PALETTE.BACKGROUND_BLUE_MAGENTA,
+  cursor: 'pointer',
+  display: 'flex',
+  paddingLeft: 16,
+  paddingTop: 8,
+  paddingBottom: 4
+})
+
+const Issue = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  paddingLeft: 16
+})
+
+const SearchInput = styled('input')({
+  appearance: 'none',
+  background: 'transparent',
+  border: 'none',
+  color: PALETTE.TEXT_MAIN,
+  fontSize: 16,
+  margin: 0,
+  paddingLeft: 0,
+  outline: 0,
+  width: '100%'
+})
+
+interface Props {
+  isEditing: boolean
+  meeting: NewJiraIssueInput_meeting
+  setIsEditing: (isEditing: boolean) => void
+  viewer: NewJiraIssueInput_viewer | null
+}
+
+const NewJiraIssueInput = (props: Props) => {
+  const {isEditing, meeting, setIsEditing, viewer} = props
+  const {id: meetingId} = meeting
+  if (!viewer) return null
+  const {id: userId, team, teamMember} = viewer
+  const {id: teamId} = team!
+  const {suggestedIntegrations} = teamMember!
+  const [newIssueText, setNewIssueText] = useState('')
+  const atmosphere = useAtmosphere()
+  const {onCompleted, onError} = useMutationProps()
+  const {items} = suggestedIntegrations
+  const suggestedIntegration = items && items[0]
+  const cloudId = suggestedIntegration?.cloudId
+  const projectKey = suggestedIntegration?.projectKey
+  const [selectedProjectKey, setSelectedProjectKey] = useState(projectKey)
+  const {originRef, menuPortal, menuProps, togglePortal, portalStatus} = useMenu(
+    MenuPosition.UPPER_RIGHT
+  )
+  const ref = useRef<HTMLInputElement>(null)
+  useEffect(() => {
+    if (portalStatus === PortalStatus.Exited) {
+      ref.current && ref.current.focus()
+    }
+  }, [portalStatus])
+
+  const handleCreateNewIssue = (e: FormEvent) => {
+    e.preventDefault()
+    if (portalStatus !== PortalStatus.Exited) return
+    setIsEditing(false)
+    if (!newIssueText.length || !selectedProjectKey || !cloudId) return
+    const variables = {
+      cloudId,
+      projectKey: selectedProjectKey,
+      summary: newIssueText,
+      teamId,
+      meetingId
+    }
+    JiraCreateIssueMutation(atmosphere, variables, {onError, onCompleted})
+    setNewIssueText('')
+  }
+
+  const handleSelectProjectKey = (projectKey: string) => {
+    setSelectedProjectKey(projectKey)
+  }
+
+  if (!isEditing) return null
+  return (
+    <>
+      <Item>
+        <Checkbox active />
+        <Issue>
+          <Form onSubmit={handleCreateNewIssue}>
+            <SearchInput
+              autoFocus
+              onBlur={handleCreateNewIssue}
+              onChange={(e) => setNewIssueText(e.target.value)}
+              placeholder='New issue summary'
+              ref={ref}
+              type='text'
+            />
+          </Form>
+          <StyledButton ref={originRef} onMouseDown={togglePortal}>
+            <StyledLink>{selectedProjectKey}</StyledLink>
+            <StyledIcon>expand_more</StyledIcon>
```

Comment:
i think this is making the line height too tall. when the icon goes away, the component should be the same height so it feels like only the copy has changed